src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,26 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
```

Comment:
Implementing Cloneable and providing a clone method is not a common practice for entities in Hibernate. Entities are usually managed by the persistence context, and the cloning of entities can lead to unexpected behavior especially with managed relationships and proxies. If you need to create copies of entities, consider using copy constructors or dedicated methods.